content/testing/functional.md

```diff
@@ -1,37 +1,31 @@
-# Functional Testing
+Functional Testing
+==================
 
 Check if a site function properly.
 
-## Two main categories:
-1. cross-browser (css/js front-end perfection)
-2. single-browser (raw functioanlity tests)
+Two main categories:
+--------------------
 
+1.	cross-browser (css/js front-end perfection)
+2.	single-browser (raw functioanlity tests)
 
-## Tools
+Tools
+-----
 
-1. Selenium
-2. Node.js based
+1.	Selenium
+2.	Node.js based
 
 ### Compare
 
-| Selenium | node.js |
-| -- | -- |
-| Tests are written in JAVA | Tests are written in nodejs |
-| Complicated to setup | Much easier to setup |
-| Tools are finalized | Exacts tools are yet to finalize |
+| Selenium | node.js | | -- | -- | | Tests are written in JAVA | Tests are written in nodejs | | Complicated to setup | Much easier to setup | | Tools are finalized | Exacts tools are yet to finalize |
 
 ### Node.js based options
 
 **List of tools**
 
-1. [phantomjs](http://phantomjs.org/) - just a browser
-2. [theintern.io](http://theintern.io/#compare) - has too many features and still not much talked about. Need to check why?
-3. [karma](http://karma-runner.github.io/) - from Angular JS team. They have one more
-4. [nightwatchjs](http://nightwatchjs.org/) - another famius tool
-
-**Evaluation criteria**
-1. Do we need unit tetsing in nodejs?
-2. Do we need cross-browser testing in nodejs?
-3. Cookie/session support
-4. File upload support
-5. AJAX support
+1.	[phantomjs](http://phantomjs.org/) - just a browser
```

Comment:
Why spaces are added here?

Please do not make unnecessary changes to line breaks and spaces